task3/app.js

```diff
@@ -0,0 +1,103 @@
+// 1. Write a JavaScript function that takes an array of numbers as input and returns
+// a new array where each element is multiplied by 2.t
+
+const func1 = (arr) => arr.map((element) => element * 2);
+
+const result1 = func1([2, 3, 4]);
+
+console.log("func1([2, 3, 4]) =", result1);
+
+// 2. Write a JavaScript function that takes an array as input and returns the length of the array.
+
+const func2 = (arr) => arr.length;
+
+const result2 = func2([2, 3, true, "a"]);
+
+console.log("func2([2, 3, true, 'a']) =", result2);
+
+// 3. Write a JavaScript function that takes an array of strings as input, sorts the strings
+// in ascending order, and returns the sorted array.
+
+const func3 = (arr) => arr.sort();
```

Comment:
FYI, be aware,  there are two methods `sort()` and `toSorted()`. Just take a look what is their difference.